src/lib/selectCylinderSets.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from 'knex';
+import { CylinderSet } from '../types/cylinderSet.types';
+import { log } from './log';
+import selectSingleCylinderSet from './selectSingleCylinderSet';
+
+export default async function selectCylinderSets(
+  trx: Knex.Transaction
+): Promise<CylinderSet[] | undefined> {
```

Comment:
```suggestion
): Promise<CylinderSet[]> {
```
Ei kai tää enää palauta `undefined`?